src/main/java/com/cburch/logisim/file/XmlWriter.java

```diff
@@ -157,7 +155,7 @@ static void sort(Node top) {
       final var a = new Node[n];
       for (int i = 0; i < n; i++) a[i] = children.item(i);
       Arrays.sort(a, nodeComparator);
-      for (int i = 0; i < n; i++) top.insertBefore(a[i], null); // moves a[i] to end
+//      for (int i = 0; i < n; i++) top.insertBefore(a[i], null); // moves a[i] to end
```

Comment:
Any reason for leaving commented out code? Please remove it or add a comment why it is commented out if you want to keep it.